lq1/default.cfg

```diff
@@ -82,19 +105,70 @@ sensitivity 3
 topcolor 11
 bottomcolor 10
 crosshair 1
+joy_enable 1
++mlook
 
 // QuakeWorld Server Infokeys
 serverinfo dq 1     // Drop Quad on death in DM
 serverinfo dr 1     // Drop Ring on death in DM
 
+// Engine-specific config Files
+// This is a bit of a hack for cross-engine support
+// Executes "default_fte.cfg" for FTE, "default_darkplaces.cfg" for DP, "default_$generic.cfg" for everything else.
+set generic darkplaces
+if "FTE" isin $qt$version$qt then set generic fte
+exec default_$generic.cfg
+
+// =============
 // Aliases
-alias 	"idkfa"		"impulse 9"
-alias 	"fullammo"	"impulse 9"
-alias	"quad"		"impulse 255"
-alias +lq-zoom90 "fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
-alias -lq-zoom90 "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90"
-alias +lq-zoom120 "fov 110;wait;fov 100;wait;fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
-alias -lq-zoom120 "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90;wait;fov 100;wait;fov 110;wait;fov 120"
+// =============
+
+// Misc
+alias 	"idkfa"		    "impulse 9"
+alias 	"fullammo"	    "impulse 9"
+alias 	"quad"		    "impulse 255"
+
+// Modern style melee attackcon_nextweapon
+alias   "+melee"        "impulse 1;+attack"
+alias   "-melee"        "-attack"
+
+// Zoom aliases
+alias   "+lq-zoom90"    "fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
+alias   "-lq-zoom90"    "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90"
+alias   "+lq-zoom120"   "fov 110;wait;fov 100;wait;fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
+alias   "-lq-zoom120"   "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90;wait;fov 100;wait;fov 110;wait;fov 120"
+
+// Dpad doom style weapon switch
+alias "+con_meleeatk"   "impulse 1;+attack;wait;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"
+alias "-con_meleeatk"   "-attack"
+alias "con_shotguns0"   "impulse 2;wait;bind UPARROW con_shotguns1;wait;echo;echo;echo;echo;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0"

```

Comment:
Why do all of these touch the other binds as well? Wouldn't it be sufficient if UPARROW binds from shotguns1 and 2 back and forth. I don't understand why the other ones need to be involved.

Reply:
if u read the melee on all it does is reset the other binds to zero

Reply:
up is shotgun and super shotgun
right is nailgun and super nailgun
left is grenade launcher and rocket launcher
down is lightning gun

Reply:
like alias "+con_meleeatk"   "impulse 1;+attack;wait

this part all it does is swap the arrow binds back to zero 
cuz other wise it will still be on one and mess with muscle memory of super shotgun super nailgun and rocket launcher being a 2x tap

;bind LEFTARROW con_explguns0;bind RIGHTARROW con_nailguns0;bind UPARROW con_shotguns0"

Reply:
I see its not to be a full toggle but only if you are on the same style of weapon otherwise always one tap is normal and two tap is super. Yeah I get it.

Reply:
![image](https://github.com/user-attachments/assets/c01d5983-6af1-4cda-aeb7-539a0cf02ed3)

u can see all the binds reset the other directions back to zero